src/components/filterSelection/FilterSelection.js

```diff
@@ -0,0 +1,13 @@
+import React, { memo } from 'react';
+
+import Box from '../box';
+
+const FilterSelection = () => {
+  return <Box>Hello FilterSelection!</Box>;
+};
+
+FilterSelection.propTypes = {};
+
+FilterSelection.defaultProps = {};
+
+export default memo(FilterSelection);
```

Comment:
Let's not memoize components by default, I know we used to do this by default using PureComponents as well but this isn't a standard thing to default to, I've said this several times before, as it can worsen performance. Build first, optimise later 😄

Reply:
Sounds like a good rule 👍